doc/sphinx/compiling.rst

```diff
@@ -32,69 +33,267 @@ Installation Prerequisites
 Linux
 -----
 
-* For Ubuntu or Debian users, the following packages should be installed using
+General Notes
+^^^^^^^^^^^^^
+
+* To download the source code, installing ``git`` is highly recommended.
+
+* SCons is only available for Python 2, so building the Python 3 module requires
+  two installations of Python (one of Python 2 and one of Python 3), even if you
+  do not intend to build the Python 2 module.
+
+* The following instructions use the system-installed versions of Python, but
+  alternate installations such as the Anaconda distribution of Python can be
+  used as well.
+
+* Cython is only required to be installed for the version of Python that also
+  has SCons installed; following the instructions below will install Cython for
+  the version of Python 2 installed in the system directories.
+
+* The SUNDIALS suite of solvers used in Cantera is available from several sources.
+  The sources in most distribution repositories may be out of date, or not
+  optimized for your hardware. The source code for SUNDIALS will be downloaded
+  and compiled with the rest of Cantera, so installing from your distribution's
+  repositories is optional.
```

Comment:
On Linux, the only problem I know of with using the system SUNDIALS is the interaction with the Matlab toolbox. On the other hand, the bundled SUNDIALS might cause problems if you wrote your own application that used both Cantera and SUNDIALS and ended up using symbols from both pieces of SUNDIALS, so neither option is perfect.

If you want to have this document just the minimal requirements, then I guess we should just not mention SUNDIALS here, otherwise there should be notes about fmt, Eigen and company here as well.